### PR TITLE
Feature/login page

### PR DESCRIPTION
**Pull request by Promit-revar**
Login page with functionalities...
Screenshot 2023-03-14 at 10 40 48 PM

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (Chetan-MckinseyBEBR) [inline public/index.html:6]: Use react fontawsome as an npm package in the project rather than CDN

File: src/components/ImageCard/ImageCard.css
Comment (Chetan-MckinseyBEBR) [inline src/components/ImageCard/ImageCard.css:1]: Why are ImageCard deleted?

File: src/components/InputComponent/index.jsx [hunk 2]
```diff
@@ -4,14 +4,16 @@ import './InputComponent.css';
 
 import propTypes from 'prop-types';
 
-export default function InputComponent({ placeholder }) {
+export default function InputComponent({ placeholder, handleChange, type }) {
   return (
     <div>
-      <input type="text" className={'login-input'} placeholder={placeholder} />
+      <input type={type} className={'login-input'} placeholder={placeholder} onChange={handleChange} />
```
Comment (Chetan-MckinseyBEBR) [inline src/components/InputComponent/index.jsx:10]: Why does the Input component has the login-input class?? That makes this component not generic

File: src/pages/LoginPage/LoginPage.css [hunk 3]
```diff
@@ -32,3 +33,14 @@
     font-size: 15px;
     font-family: 'Inter';
 }
+.error{
```
Comment (Chetan-MckinseyBEBR) [inline src/pages/LoginPage/LoginPage.css:36]: empty lines after each code section

File: src/utils/authMakeRequest/index.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ const makeRequest = async (apiEndPoint, dynamicConfig = {}) => {
     method: apiEndPoint.method,
     ...dynamicConfig,
   };
-  console.log('requestDetails', requestDetails);
+  //console.log('requestDetails', requestDetails);
```
Comment (Chetan-MckinseyBEBR) [inline src/utils/authMakeRequest/index.js:11]: Remove commented code

File: src/utils/makeRequest/index.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 import { BACKEND_URL } from '../../constants/apiEndpoints';
-import AUTHORIZATION_TOKEN from '../../constants/authorizationToken';
+import AUTHORIZATION_TOKEN from '../../constants/authorizationToken.js';
```
Comment (Chetan-MckinseyBEBR) [inline src/utils/makeRequest/index.js:3]: Why .js?

**Review by Chetan-MckinseyBEBR — CHANGES_REQUESTED** [review]
Added few comments

File: src/components/ImageCard/ImageCard.css
Comment (Promit-revar) [reply src/components/ImageCard/ImageCard.css:1]: One of my teammate told me that we are not using it anymore but I will add it again

File: src/components/InputComponent/index.jsx (hunk 2 above)
Comment (Promit-revar) [reply src/components/InputComponent/index.jsx:10]: its because no where else we are using input component other than login

**Comment by Promit-revar** [comment]
Screenshot 2023-03-16 at 11 33 56 AM

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

**Pull request merged**